src/utils/template-helper.ts

```diff
@@ -280,6 +280,9 @@ export class TemplateHelper {
      * @returns {boolean} True if the property is an enumeration with entity values property.
      */
     isEnumPropertyWithEntityValues(property: Property) {
+        if (property.characteristic instanceof DefaultEnumeration && property.characteristic.values?.[0] instanceof DefaultEntityInstance) {
+            return false;
+        }
```

Comment:
this part  if (property.characteristic instanceof DefaultEnumeration && property.characteristic.values?.[0] instanceof DefaultEntityInstance) {
            return false;
        }  should be removed